pkg/apis/imageregistry/v1/types.go

```diff
@@ -169,12 +170,17 @@ type ImageRegistryConfigStorageFilesystem struct {
 	VolumeSource corev1.VolumeSource `json:"volumeSource,omitempty"`
 }
 
+type ImageRegistryConfigStoragePVC struct {
+	Size resource.Quantity `json:"size,omitempty"`
```

Comment:
what's the behavior if/when a user updates this value?

Reply:
@bparees You can not change it. Otherwise, we will need to re-create PVC and you will lose the current data since another PV is likely to be selected.

Reply:
yes, this is the same issue we discussed extensively around how to deal w/ s3 bucket changes.  Ultimately (After a few false starts) we decided to let people do it because the alternatives are not great (you install a new cluster, we gave you a 100gig PVC by default. You know that's not going to be enough, so you want to change it...what's the process for doing so?  Basically I have to remove the registry, disable the operator, delete the CR, create a new CR w/ the right storage value, and then re-enable the operator... or something along those lines, right?)


In this case, perhaps a better experience would be to not allow them to specify the size at all, just require them to provide the name of the PVC they want to use.  If they provide no PVC name, we create a PVC w/ a size we choose (as you are doing).  If they change the PVC name in the future, just like s3, they lose their storage, that was their choice.

That is similar to the current S3 experience.

Reply:
Why do we need to ask the PVC name? As you told before, its our own namespace and there is no need to have more than one PVC in our namespace.